### PR TITLE
cluster-api-provider-aws: use multistep job for e2e-upgrade

### PR DESCRIPTION
**Pull request by vrutkovs**
Run AWS upgrade using multistep, persist metrics and store container logs using Loki

### CODE REVIEW

File: ci-operator/config/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-master.yaml [hunk 1]
```diff
@@ -63,11 +63,10 @@ tests:
   openshift_installer_custom_test_image:
     cluster_profile: aws
     from: stable:cluster-api-actuator-pkg
-- as: e2e-aws-upgrade
-  commands: TEST_SUITE=all run-upgrade-tests
-  openshift_installer:
+- as: e2e-upgrade
```
Comment (michaelgugino) [inline ci-operator/config/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-master.yaml:66]: Is there a reason to remove the 'aws' part of this job name?  With aws in the name, it makes it more clear on PRs what is running where.  This repo should never, for instance, run an e2e job on GCP.

Comment (vrutkovs) [reply ci-operator/config/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-master.yaml]: Makes sense, reverted the rename

**Comment by vrutkovs** [comment]
4.4 still flaking

/retest

**Comment by vrutkovs** [comment]
/retest

4.4 is still flaking

**Review by michaelgugino — APPROVED** [review]
/lgtm

**Pull request merged**